luatest/server.lua

```diff
@@ -232,13 +232,20 @@ function Server:start(opts)
         table.insert(log_cmd, string.format('%s=%q', k, v))
         env[k] = v
     end
+    table.insert(log_cmd, arg[-1])
     table.insert(log_cmd, self.command)
     for _, v in ipairs(self.args) do
         table.insert(log_cmd, string.format('%q', v))
     end
     log.debug(table.concat(log_cmd, ' '))
 
-    self.process = Process:start(self.command, self.args, env, {
```

Comment:
Let's also remove the X bit and the sha-bang (`#!/usr/bin/env tarantool`) from `luatest/server_instance.lua`?

Reply:
Fixed.